SSA/Projects/InstCombine/Test.lean

```diff
@@ -64,12 +86,42 @@ def opRet : Op 0 := [mlir_op| "llvm.return"(%4) : (i32) -> ()]
   I've changed them to be consistent with how the current code works,
   please check that the tested behaviour is actually the desired behaviour
 -/
-#eval mkExpr    (Γn 1) op0    ["arg0"]
-#eval mkExpr    (Γn 2) op1    ["0", "arg0"]
-#eval mkExpr    (Γn 3) op2    ["1", "0", "arg0"]
-#eval mkExpr    (Γn 4) op3    ["2", "1", "0", "arg0"]
-#eval mkExpr    (Γn 5) op4    ["3", "2", "1", "0", "arg0"]
-#eval mkReturn  (Γn 6) opRet  ["4", "3", "2", "1", "0", "arg0"]
+
+/--
+info: Except.ok ⟨EffectKind.pure, ⟨i32, InstCombine.MOp.const (ConcreteOrMVar.concrete 32) 8[[]]⟩⟩
+-/
+#guard_msgs in #eval mkExpr    (Γn 1) op0    ["arg0"]
+
+/--
+info: Except.ok ⟨EffectKind.pure, ⟨i32, InstCombine.MOp.const (ConcreteOrMVar.concrete 32) 31[[]]⟩⟩
+-/
+#guard_msgs in #eval mkExpr    (Γn 2) op1    ["0", "arg0"]
+
+/--
+info: Except.ok ⟨EffectKind.pure, ⟨i32, InstCombine.MOp.binary
+    (ConcreteOrMVar.concrete 32)
+    (InstCombine.MOp.BinaryOp.ashr)[[%2, ,, %0]]⟩⟩
+    -/
+#guard_msgs in #eval mkExpr    (Γn 3) op2    ["1", "0", "arg0"]
+
+/--
+info: Except.ok ⟨EffectKind.pure, ⟨i32, InstCombine.MOp.binary
+    (ConcreteOrMVar.concrete 32)
+    (InstCombine.MOp.BinaryOp.and)[[%0, ,, %2]]⟩⟩
+-/
+#guard_msgs in #eval mkExpr    (Γn 4) op3    ["2", "1", "0", "arg0"]
+
+/--
+info: Except.ok ⟨EffectKind.pure, ⟨i32, InstCombine.MOp.binary
+    (ConcreteOrMVar.concrete 32)
+    (InstCombine.MOp.BinaryOp.add)[[%0, ,, %1]]⟩⟩
```

Comment:
Interestingly, here the printer seems to be broken as it yields a `, ,,`. I suggest to commit the broken guard_msg and to open an issue for this.